validator_client/src/cli.rs

```diff
@@ -23,7 +22,6 @@ pub fn cli_app<'a, 'b>() -> App<'a, 'b> {
                 .long("beacon-nodes")
                 .value_name("NETWORK_ADDRESSES")
                 .help("Comma-separated addresses to one or more beacon node HTTP APIs")
```

Comment:
I think we should probably mention the default here. E.g., `"Comma-separated addresses to one or more beacon node HTTP APIs". Default is http://localhost:5052.`